app/models/service_ansible_playbook.rb

```diff
@@ -68,12 +73,16 @@ def on_error(action)
 
   private
 
+  def retain_resources_on_retirement?
```

Comment:
If we end up calling this from the task it will need to be a public method.  Sorry.